.travis.yml

```diff
@@ -1,13 +1,32 @@
 language: cpp
 compiler: gcc
+sudo: require
+dist: trusty
 
-dist: xenial
-addons:
-  apt:
-    packages:
-    - libboost-dev
-    - qt5-default
+before_install:
```

Comment:
Could you stick with using the `xenial` environment instead of `trusty` and using `addons` for installing dependencies?

Reply:
No. Trusty is currently the oldest still-supported release. The AppImage will run on _newer_ but not on _older_ distributions than what it was built on. Hence, we recommend to build on the oldest still-supported release

Reply:
> Regarding the installing, I don't think this works with PPAs. Hence I do it like I do it, to get a recent Qt for trusty.

I does for those in Travis's approved list and this particular one is. The point of using `addons` is that we can get rid of using `sudo` and hence get a much faster containerized buid.

Reply:
Didn't know that there are whitelisted ppas. Thanks for the hint, will change it.